### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by teobouvard**

### CODE REVIEW

File: src/building/bootstrapping.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ Stage N `std` is pretty much necessary for any useful work with the stage N comp
 Without it, you can only compile programs with `#![no_core]` -- not terribly useful!
 
 The reason these need to be different is because they aren't necessarily ABI-compatible:
-there could be a new layout optimizations, changes to MIR, or other changes
+there could be new layout optimizations, changes to the MIR, or other changes
```
Comment (camelid) [inline src/building/bootstrapping.md:221]: I think it's correct without the "the".

Comment (teobouvard) [reply src/building/bootstrapping.md:221]: I was originally only fixing the "a", but found that adding "the" in the following sounded nicer. As you prefer !

Comment (camelid) [reply src/building/bootstrapping.md:221]: Not a big deal either way, but I think it's slightly more standard to do it without the "the".

Comment (teobouvard) [reply src/building/bootstrapping.md:221]: I'll remove it then

**Review by camelid — APPROVED** [review]
Thanks!

**Pull request merged**